.github/workflows/go.yml

```diff
@@ -38,9 +38,4 @@ jobs:
       - name: Lint
         run: |
           export PATH=$PATH:~/go/bin
-          ./bin/golangci-lint run --deadline=10m \
-            --disable-all --enable govet \
-            --enable staticcheck --enable gosimple \
-            --enable unconvert --enable ineffassign \
-            --enable revive --enable goimports \
-            --enable unparam
\ No newline at end of file
+          ./bin/golangci-lint run
```

Comment:
`./run_tests.sh` does this and more, it's better to invoke that script here instead:
```
- name: Test and lint
  run: |
    ./run_tests.sh
```